fitbenchmarking/fitbenchmarking
# Minor updates to documentation

#### Description of Work
This PR makes some minor updates to the documentation.

Fixes #974
Fixes #973 

#### Testing Instructions

1. Read the docs and make sure they make sense

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
